content/api-reference/index.textile

```diff
@@ -0,0 +1,21 @@
+---
+title: API Reference
+meta_description: "API Reference section of the Ably developer documentation."
+meta_keywords: "Ably, Ably realtime, API Reference, Realtime SDK, REST SDK, REST API"
+section: api-reference
+index: 0
+---
+
+The API references for the Ably Platform. 
```

Comment:
Good point. I'll add some more detail. 

On the question of the website view, yes, it would look like this:

```
      {
        title: 'API Reference',
        items: [
          { text: 'Overview', url: documentation_path(page, 'api-reference') },
          { text: 'Realtime SDK', url: documentation_path(page, 'api-reference/realtime-sdk') },
          { text: 'REST SDK', url: documentation_path(page, 'api-reference/rest-sdk') },
          { text: 'REST API', url: documentation_path(page, 'api-reference/rest-api') }
        ]
      },
```